src/Root/rootSaga.js

```diff
@@ -0,0 +1,6 @@
+// import { fork } from 'redux-saga/effects';
+
+export default function* root() {
```

Comment:
I just found that there is `` import rootSaga from './Root/rootSaga'; `` in the `` src/index.js `` file. Do you think it'd be a good idea to rename `` root `` to `` rootSaga ``?